gunicorn_config.py

```diff
@@ -10,7 +10,6 @@
 workers = 4
 worker_class = "eventlet"
 worker_connections = 256
-errorlog = "/home/vcap/logs/gunicorn_error.log"
```

Comment:
I think it is worth calling out that this will also change things for the PaaS deployed version and stop putting logs into the file (and then those will no longer be shipped to (probably Cloudwatch)).

Sakis pointed out that we pretty much never get any useful logs from gunicorn though so maybe this is tolerable to turn it off for the PaaS.

Alternatively, we could maybe try and override this by either

- keeping this line in the file and change the entrypoint.sh file to explicitely set `errorlog -` so that it goes to stdout
- removing this line in the file and change the command that is used to run the PaaS so that it specifies the file should be used for errorlog.

I'm not really sure what I think. @sakisv do you have an opinion?

Reply:
Personally I don't see any value in having these logs, and I would remove them (and also remove the relevant clouwatch-agent config bit from the run-app-paas script).

But, before doing that I'd like to confirm with the dev team that these logs are indeed useless and they haven't looked at them.